lambdas/service/app.py

```diff
@@ -902,6 +905,11 @@ def filter_schema(relation: str) -> JSON:
 
 
 def repository_search_params_spec():
+    # This spec is shared across all entity types, so no single value would be
+    # correct here. Using the largest possible value ensures that we don't
+    # incorrectly reject valid requests, at the cost of accepting some invalid
+    # ones.
+    max_page_size = max(app.metadata_plugin.page_size_limits.values())
```

Comment:
This using the default catalog and ignores the others. But I think we have this problem elsewhere. If you end up leaving this approach in, add a FIXME referring to https://github.com/DataBiosphere/azul/issues/4288.

But see my comment above.

Reply:
But most importantly, you have to document the quirks to the Azul *user*, not to the *developer*. The right place for that is the description of the actual parameter.